crates/cext/cbindgen.toml

```diff
@@ -11,6 +12,11 @@ after_includes = """
 
 // Complex number typedefs -- note these are memory aligned but
 // not calling convention compatible.
+#ifdef __cplusplus
+typedef std::complex<float> QkComplex32;
```

Comment:
I believe `std::complex` is defined in the `<complex>` header, but right now we always include `<complex.h>` (see  `sys_includes` on L7). Maybe we should remove the `sys_includes` and instead have something like this?
```toml
#ifdef __cplusplus
#include <complex>
C++ typedefs go here ...
#else
#include <complex.h>
C typedefs go here...
#endif
```